.github/workflows/nodejs.yml

```diff
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
I figured we'd want to test in the latest Node version as well? I can remove if you have reasons for only 10 & 12

Reply:
iirc node 14 wasn't stable enough when I first added this, but yeah, should test that as well :+1: